docker/packager/packager

```diff
@@ -276,10 +276,7 @@ def parse_env_variables(
         if is_release_build(debug_build, package_type, sanitizer, coverage):
             cmake_flags.append("-DSPLIT_DEBUG_SYMBOLS=ON")
             result.append("WITH_PERFORMANCE=1")
-            if is_cross_arm:
-                cmake_flags.append("-DBUILD_STANDALONE_KEEPER=1")
-            else:
-                result.append("BUILD_MUSL_KEEPER=1")
+            cmake_flags.append("-DBUILD_STANDALONE_KEEPER=1")
```

Comment:
Now that the standalone build is gone, we can also remove this CMake option? There are some occurrences of `BUILD_STANDALONE_KEEPER` in CMakeFiles and scripts and we can get rid of all of them.

Reply:
There is still a difference where we build a separate binary vs just generating a symlink so the option still makes sense.